force-app/main/default/pages/CON_DeleteContactOverride.page

```diff
@@ -1,58 +1,83 @@
 <apex:page standardController="Contact" extensions="CON_DeleteContactOverride_CTRL" action="{!processDelete}" id="delContactPage">
     <apex:slds />
+    <apex:includeLightning />
+
+    <script>
+        if ({!NOT(hasMinimumAccess)}) {
+           $Lightning.use("{!namespace}" + ":RD2_EnablementApp", function() {
```

Comment:
It seems a little weird to have a reference to the RD2_EnablementApp here in the Contact Delete Override page. I suppose it makes sense to not introduce another Aura Application when we already have one that has utilIllustration declared as a dependency.

Reply:
I'd say let's address in a different WI if need be. It's not a huge deal, but is still sorta awkward.